client/src/__tests__/unit-tests/driver/BitbakeESDK.test.ts

```diff
@@ -102,11 +104,11 @@ describe('Bitbake ESDK Test Suite', () => {
     loadJsonMock.mockReturnValueOnce(originalConfig)
     const saveJsonMock = jest.spyOn(JSONFile, 'saveJsonFile').mockImplementation(() => {})
 
-    const bitBakeProjectScannerMock = { resolveContainerPath: jest.fn().mockImplementation((arg) => arg) } as any
-    const clientMock = jest.fn() as any
-    const getVariableValueMock = jest.spyOn(LanguageClient, 'getVariableValue')
+    const bitBakeProjectScannerMock = { resolveContainerPath: jest.fn().mockImplementation((arg) => arg) } as unknown as BitBakeProjectScanner
+    const clientMock = jest.fn() as unknown as LanguageClient
```

Comment:
It is funny to see a trick like this, but as long as it works and the test doesn't break.

Reply:
Yup. In a way this is the same as `any`, but I got tired trying to find alternatives, and I thought it could be justified since it is just a mock in our tests. 

I remember reading the `reinterpret_cast` syntax in C++ is ugly by design so people will think twice before using it. So I thought maybe this is still better than allowing `any`.